packages/rust-bindings/Cargo.toml

```diff
@@ -48,7 +51,7 @@ version = "0"
 [dependencies.concordium-contracts-common]
 features = ["derive-serde"]
 path = "../../deps/concordium-base/concordium-contracts-common/concordium-contracts-common"
-version = "3.1.0"
+version = "4.0.0"
```

Comment:
And please update the version in the `package.json` for this package (and update the CHANGELOG)